server/src/messageHandler.mjs

```diff
@@ -345,6 +345,10 @@ async function handleMessage(message, userId, ws) {
   }
   const dly = stateManagement.getAppropriateDelay(userId, oMsg.method);
   await util.delay(dly);
+
+  const userData = userManagement.getWsForUser(config?.dotConfig?.interactionService?.user);
+  util.createInteractionLog(finalResponse, JSON.parse(message).method, JSON.parse(message).params, userData); // creating interaction log and send it to the client
```

Comment:
this condition is placed inside `createInteractionLog` the function, so instead of having it in all the places where it is being called, I thought to put it inside the function itself